docs/src/modules/java/partials/actions.adoc

```diff
@@ -1,25 +1,19 @@
 include::ROOT:partial$include.adoc[]
 
-Actions are stateless functions that can be triggered in multiple ways. For example, by
+Actions are stateless functions that can be used to implement different uses cases, such as:
+
+- a pure function
+- request conversion - you can use Actions to convert incoming data into a different format before forwarding a call to a different component.
+- subscribing to events within the same service
```

Comment:
some mention that it can publish events (to a different service)

Reply:
Good point, that makes me wondering if we should simply say: Publish and Subscribe to events without making a distinction between internal and external events. 

Then, when move to the "Publish and Subscribe" page, we can explain that we can subscribe to local journal or to topic, push events to another service or to another topic, etc. 

At the end, the use cases around publishing and subscribing are the same except the source of the subscriptions. 

So it may make sense to explain them together.

Reply:
makes sense in this overview